src/components/Profile.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Profile = () => (
+  <>
+    <div>
+      <h2>Our Profile</h2>
+    </div>
+  </>
+);
+
```

Comment:
- as part of the requirements The My Profile section should display all reserved rockets and space missions. currently its empty so kindly implement that requirements 🙏